### ISSUE TITLE
Automate primaryClusterEndPoint configuration in multicluster CIS

### ISSUE THREAD

**Issue by avinashchundu9** [F5Networks/k8s-bigip-ctlr#3354]
#### Title
Automate primaryClusterEndPoint configuration in multicluster CIS

#### Description
In a multi-kubernetes cluster where there is no direct pod-to-pod communication between the clusters, configuring primaryClusterEndPoint is either a manual load balancer or NodePort. Node port is not an option in the prod environment and the manual load balancer is hard to manage when we have multiple CIS and dynamic worker nodes.

#### Actual Problem
primaryClusterEndPoint endpoint is needed for health checks between the CIS. We need a CIS to configure and manage primaryClusterEndPoint in the cluster.

#### Solution Proposed
Request inputs for creating primaryClusterEndPoint load balancer and create it during CIS startup. 

Example: Request IP address, and service details in config map and use those details to create a load balancer for primaryClusterEndPoint during CIS startup.

#### Alternatives
Allow users to define multiple NodePort endpoints for primaryClusterEndPoint. This will enable users who want to stick with Nodeport.

**Comment by trinaths** [comment]
Created [CONTCNTR-4664] for internal tracking.

**Comment by alonsocamaro** [comment]
Couldn't we just use Kubernetes API to check if the Primary CIS is Ready?

This would simplify things greatly

**Comment by avinashchundu9** [comment]
Yes, that's even better solution since we are already providing kubeconfig as secrets in the configmaps.
